package/gargoyle/files/www/js/access.js

```diff
@@ -591,40 +576,43 @@ function updateVisibility()
 		remoteHttpsElement.value = "";
 		remoteHttpElement.value = "";
 	}
-	document.getElementById("noip6redi").style.display = remoteWebProtocol != "disabled" ? "block" : "none";
+	document.getElementById("noip6redi").style.display = remoteWebProtocol == "disabled" || (remoteWebProtocol == "https" && localWebProtocol == "redirect") ? "none" : "";
```

Comment:
Please use "block" and "none".
Setting to "" instead of "block" will not override any styles that may be applied by other CSS (i.e. not directly to the element) potentially leading to less defined behaviour. This is also more consistent with the remainder of Gargoyle

Reply:
Will do. At some places I use `inline-block` in CSS. Do I use `"inline-block"` then in JS or stick to `""`?

Reply:
Yes use the appropriate display setting as needed